.github/actions/monitor-verified-batches/action.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: monitor-verified-batches
+description: Check that batches are being verified in a CDK environment
+
+inputs:
+  verified_batches_target:
+    description: The minimum number of batches to be verified
+    required: false
+    default: '30'
+  timeout:
+    description: The script timeout in seconds
+    required: false
+    default: '600' # 10 minutes
+
+runs:
+  using: "composite"
+  steps:
+    - name: Check that batches are being verified
+      working-directory: .github/actions/monitor-verified-batches
+      shell: bash
+      run: ./batch_verification_monitor.sh ${{ inputs.verified_batches_target }} ${{ inputs.timeout }}
```

Comment:
wondering if it may be useful to just have a shared git action we can reference all over our code base, see we're using this verification monitor logic quite often..

Reply:
yeah that's the plan, for the moment we can use a composite action that we reference in all of our workflows in this repo